tools/setup_helpers/cmake.py

```diff
@@ -349,4 +349,6 @@ def build(self, my_env):
             build_args += ['--', '/p:CL_MPCount={}'.format(max_jobs)]
         else:
             build_args += ['--', '-j', max_jobs]
+        if USE_NINJA:
+            build_args += ['-v']
```

Comment:
Why is this needed?

Reply:
Comparing the `binary_linux_libtorch_3_7m_cu111_gcc5_4_cxx11-abi_nightly_shared-with-deps_build` run from this PR and yesterday's nightly, we have:
[this PR](https://app.circleci.com/pipelines/github/pytorch/pytorch/327301/workflows/a93b2038-abd9-4b71-8dc5-31362720b7a1/jobs/13711481) with 5235 lines and
[nightly](https://app.circleci.com/pipelines/github/pytorch/pytorch/327692/workflows/395092c6-513a-47a2-b5d4-1e99274c554b/jobs/13730256) with 5442 lines

Reply:
it should be fine.